### ISSUE TITLE
Why is the code using a nested namespace OneSignal.CSharp.SDK ?

### ISSUE THREAD

**Issue by AlfonsoML** [mundipagg/OneSignal-CSharp-SDK#17]
Is there any reason to use everything with a nested namespace like OneSignal.CSharp.SDK.OneSignalClient instead of using just OneSignal.OneSignalClient 

Honestly I don't see any reason to keep the language in the namespace, nor the "SDK" part.

I would suggest to simplify all the usages and provide a major version bump to notify of the big change, so the sooner that it's done, the better.

Would you accept such pull request?

**Comment by mvrmoreira** [comment]
I totally agree with you, of course if you send a pull request we will accept it ;)
